gen_cert.py

```diff
@@ -1804,7 +1797,7 @@ def fontlist_with_style(a_style):
         #   * honor code url at the bottom
 
         # SECTION: Issued Date
-        date_string = u"{0}".format(get_cert_date(generate_date, self.issued_date))
+        date_string = u"{0}".format(get_cert_date(generate_date, self.issued_date, self.date_format, self.locale))
```

Comment:
I know this was already there, but why are we doing a string format when there's only a single value?

Seems superfluous...